docker-compose.yml

```diff
@@ -57,13 +58,12 @@ services:
     depends_on:
       - db
       - redis
-#      - es
+    #      - es
     volumes:
       - ./public/system:/mastodon/public/system
 
   streaming:
     build: .
-    image: dsterry/ecko
```

Comment:
@weex I removed these as well, Do you want me to put this in the same docker image or would you like it to be from the source only?

Reply:
Thinking of what's easiest for a new installation, I would prefer to keep the `image:` line in.